src/Compilers/CSharp/Portable/Symbols/Synthesized/Records/SynthesizedRecordPropertySymbol.cs

```diff
@@ -243,87 +261,13 @@ private sealed class InitAccessorSymbol : SynthesizedInstanceMethodSymbol
 
             public override MethodKind MethodKind => MethodKind.PropertySet;
 
-            public override int Arity => 0;
-
-            public override bool IsExtensionMethod => false;
-
-            public override bool HidesBaseMethodsByName => false;
-
-            public override bool IsVararg => false;
-
-            public override bool ReturnsVoid => true;
-
-            public override bool IsAsync => false;
-
-            public override RefKind RefKind => RefKind.None;
-
-            public override FlowAnalysisAnnotations ReturnTypeFlowAnalysisAnnotations => FlowAnalysisAnnotations.None;
-
-            public override ImmutableArray<TypeWithAnnotations> TypeArgumentsWithAnnotations => ImmutableArray<TypeWithAnnotations>.Empty;
-
-            public override ImmutableArray<TypeParameterSymbol> TypeParameters => ImmutableArray<TypeParameterSymbol>.Empty;
-
             public override ImmutableArray<ParameterSymbol> Parameters => ImmutableArray.Create(SynthesizedParameterSymbol.Create(
```

Comment:
>SynthesizedParameterSymbol.Create [](start = 96, length = 33)

Every time allocating a new parameter.

Reply:
This was not part of this PR and has changed in [https://github.com/dotnet/roslyn/pull/44883](https://github.com/dotnet/roslyn/pull/44883).

---
In reply to: [443960042](https://github.com/dotnet/roslyn/pull/45336#discussion_r443960042) [](ancestors = 443960042)